### PR TITLE
[fix]:Optimize the quick start module used in vue3(CHINESE)

### PR DESCRIPTION
**Pull request by piexlmax**
Does not provide an export named'default'will be used in vue3, but it is not explained in the document. And the quotation at the beginning of the document is not clear.

### CODE REVIEW

File: docs/zh/guide/index.md [hunk 1]
```diff
@@ -34,6 +34,9 @@
 ## JavaScript
 
 ```js
+import VueRouter from "vue-router"
+// 在vue3中你或许需要 import {createRouter,createWebHashHistory} from "vue-router"
```
Comment (edison1105) [inline docs/zh/guide/index.md:38]:
```suggestion
// 在 vue3 中你或许需要 import { createRouter,createWebHashHistory } from "vue-router"
```

File: docs/zh/guide/index.md [hunk 2]
```diff
@@ -55,6 +58,17 @@ const router = VueRouter.createRouter({
   history: VueRouter.createWebHashHistory(),
   routes, // `routes: routes` 的缩写
 })
+ 
+// 在vue3中使用时候会出现 The requested module '/node_modules/.vite/vue-router.js?v=289caafc' does not provide an export named 'default' 的错误，这时你不可以使用 import VueRouter from "vue-router"
```
Comment (edison1105) [inline docs/zh/guide/index.md:62]: `在 vue3 中`

一般中英文混写的时候，我们习惯英文单词前后加一个空格。

File: docs/zh/guide/index.md [hunk 3]
```diff
@@ -55,6 +58,17 @@ const router = VueRouter.createRouter({
...
+// 正确的做法为
+/*
+import {createRouter,createWebHashHistory} from "vue-router"
```
Comment (edison1105) [inline docs/zh/guide/index.md:65]:
```suggestion
import { createRouter,createWebHashHistory } from "vue-router"
```

**Review by edison1105 — CHANGES_REQUESTED** [review]

**Comment by posva** [comment]
Thank you for the feed-back. We will find a different way

**Pull request closed without merging**